node-db/week-13/homework.md

```diff
@@ -1,8 +1,5 @@
 # Homework
 
-1. Finish the tasks from the workshop
-1. Try adding an `/about` route to your app to serve some HTML. How is this different from using express's static middleware? You can see one way of doing it [here](http://stackoverflow.com/questions/4529586/render-basic-html-view)
-1. Read & Practice: [Express Middleware](http://expressjs.com/en/guide/using-middleware.html) - Be prepared to name & describe at least 1 middleware at the class
-1. Learn about [Templating Engines](https://expressjs.com/en/guide/using-template-engines.html) and make a simple page with [Handlebars](http://handlebarsjs.com/) (with or without Express), and create a repository and post the link of your project on your card
-1. (Optional) Complete the nodeschool.io
-   [learnyounode tutorial](https://github.com/workshopper/learnyounode)
+1. Finish the tasks (1 to 7) from the workshop
+2. Finish at least challenge one from the following challenge. 
```

Comment:
Is the homework only task 1 to 7? Is the React bit not a part of it ... in all cases, I think the second homework would be too much for this week